staging/src/k8s.io/pod-security-admission/policy/helpers.go

```diff
@@ -16,7 +16,23 @@ limitations under the License.
 
 package policy
 
-import "strings"
+import (
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"strings"
+)
+
+var (
```

Comment:
How about creating a `paths.go` file with all well known paths ?

Reply:
That's a good idea